apps/web/src/components/Copyable.tsx

```diff
@@ -0,0 +1,30 @@
+import type { FC } from "react";
+import React from "react";
+
+import { CopyToClipboard } from "./CopyToClipboard";
+import { Link } from "./Link";
+
+export interface CopyableProps {
+  href?: string;
+  value: string;
+  tooltipText?: string;
+  children?: React.ReactNode;
+}
+
+export const Copyable: FC<CopyableProps> = ({
+  href,
```

Comment:
I'd prefer to remove this prop and keep `Copyable` as generic as possible. If you want to render a link then pass it down but don't as `children`

Reply:
Done!